otcextensions/sdk/cce/v3/cluster_node.py

```diff
@@ -47,6 +47,43 @@ class PublicIPSpec(resource.Resource):
     floating_ip = resource.Body('eip', type=dict)
```

Comment:
pls look at 41-46, may be we can uncomment bandwidth?

Reply:
Nope, external IP is something different. It is the management IP to use kubectl to manage the cluster resources. For ECS and in this case cce node the name floating_ip seems to be the common name for it: https://opendev.org/openstack/openstacksdk/src/branch/master/openstack/network/v2/floating_ip.py

Reply:
I can't find "bandwidth" anywhere in the document.